### PR TITLE
Fix running cypress jobs locally

### PR DESCRIPTION
**Pull request by connorwstein**
Default port was incorrect and the keystore is needed to have eth so submit txes. With these changes, you can follow the steps in integration/README.md to run the tests locally.

Also added a check to prevent specifying a funding key address as a from address, without which you would silently not submit transactions if you did specify that fromAddress.

### CODE REVIEW

File: tools/docker/docker-compose.yaml [hunk 1]
```diff
@@ -7,6 +7,13 @@ services:
     build:
       context: ../../
       dockerfile: core/chainlink.Dockerfile
+    # Note that the keystore import allows us to submit transactions
+    # because addresses are specified when starting the parity/geth node
+    # to be prefunded with eth. Use the "entrypoint" command below instead of
+    # the existing command if you want to run cypress tests locally.
+    # TODO: once FM supports multiple keys we should be able to use
```
Comment (samsondav) [inline tools/docker/docker-compose.yaml:14]: FM does support multiple keys, with key pinning

File: tools/docker/docker-compose.yaml [hunk 2]
```diff
@@ -40,6 +47,7 @@ services:
     build:
       context: ../../
       dockerfile: core/chainlink.Dockerfile
+    # entrypoint: /bin/sh -c "chainlink node import /run/secrets/keystore && chainlink node start -d -p /run/secrets/node_password -a /run/secrets/apicredentials"
```
Comment (samsondav) [inline tools/docker/docker-compose.yaml:50]: commented?

**Review by samsondav — CHANGES_REQUESTED** [review]

File: tools/docker/docker-compose.yaml (hunk 2 above)
Comment (connorwstein) [reply tools/docker/docker-compose.yaml:50]: yah - to be used if you want to run the tests locally

File: core/services/fluxmonitor/flux_monitor.go [hunk 3]
```diff
@@ -654,15 +654,15 @@ func (p *PollingDeviationChecker) SetOracleAddress() error {
 	accounts := p.store.KeyStore.Accounts()
 	for _, acct := range accounts {
 		for _, oracleAddr := range oracleAddrs {
-			if acct.Address == oracleAddr {
+			if acct.Address.Hex() == oracleAddr.Hex() {
```
Comment (samsondav) [inline core/services/fluxmonitor/flux_monitor.go:657]: Was this broken??

File: tools/ci-ts/tests/flux-monitor.test.ts [hunk 4]
```diff
@@ -215,6 +217,7 @@ describe('FluxMonitor / FluxAggregator integration with two nodes', () => {
     await linkToken.transfer(fluxAggregator.address, deposit).then(t.txWait)
     await fluxAggregator.updateAvailableFunds().then(t.txWait)
 
+    console.log(await fluxAggregator.getOracles())
```
Comment (samsondav) [inline tools/ci-ts/tests/flux-monitor.test.ts:220]: Still necessary?

File: core/services/fluxmonitor/flux_monitor.go (hunk 3 above)
Comment (samsondav) [reply core/services/fluxmonitor/flux_monitor.go:657]: .Equal?

**Review by samsondav — APPROVED** [review]

File: tools/ci-ts/tests/flux-monitor.test.ts (hunk 4 above)
Comment (connorwstein) [reply tools/ci-ts/tests/flux-monitor.test.ts:220]: no just helpful logs

File: core/services/fluxmonitor/flux_monitor.go (hunk 3 above)
Comment (connorwstein) [reply core/services/fluxmonitor/flux_monitor.go:657]: no just precarious - the type doesn't have an equal

**Review by samsondav — APPROVED** [review]

**Pull request merged**